docs/email_style.md

```diff
@@ -11,7 +11,7 @@
 5. Remove hashtags
 6. We are amplifying calls to defund the police, not reform them. The core message should be a direct statement to defund the police. Specific demands for reforms (e.g., policing tactics and training) will be removed.
 7. Make the email specific to your city/county. Please include sources for this information. Some suggestions:
-    - Can you include dollar amounts or percentages of how much of the budget goes to the police, in comparison to other community programs, initiatives, or infrastructure? 
+    - Can you include dollar amounts or percentages of how much of the budget goes to the police, in comparison to other community programs, initiatives, or infrastructure?
```

Comment:
Can get rid of this churn if you want, that's my editor removing trailing spaces...